### ISSUE TITLE
Vidstreaming/multiquality download option in GOGO

### ISSUE THREAD

**Issue by athish0** [saikou-app/saikou#829]
### Summary

It would be better if vidstreaming or multiquality server's download option that is available in gogoanime be available here.

### The Problem

Download options for GOGO (Vidstreaming/multiquality)

### The Ideal Solution

It would be great if download button is added.

**Comment by Adolar0042** [comment]
It unfortunately isn't as simple as adding a download button, the entire logic behind downloading multi quality streams is yet to be made. In the meantime I recommend you use third party download managers such as 1DM or ADM and pick them as your preferred download manager in the settings of Saikou to easily download multi quality streams.

**Comment by athish0** [comment]
Yes, i know it's not that easy. But if you're working on it then it's great!

**Comment by Adolar0042** [comment]
unfortunately to my knowledge no one is currently working on an in-app downloader for multi quality streams, but it is a highly requested feature so it's likely to be added at some point in the future.

**Comment by athish0** [comment]
It's quite important to me cause, i have limited internet access and use gogo streams as they have encoded low size episodes. And it's more better to stream it offline. Hence the feature would help a lot. So I'll reopen the issue till it's added😊
